src/extension/inspire/src/test/java/org/geoserver/inspire/wms/WMSExtendedCapabilitiesTest.java

```diff
@@ -28,7 +27,7 @@ public void testExtendedCaps() throws Exception {
         getGeoServer().save(wms);
 
         Document dom = getAsDOM("wms?request=getcapabilities");
-        assertEquals(NAMESPACE, dom.getDocumentElement().getAttribute("xmlns:inspire_vs"));
+        assertEquals(VS_NAMESPACE, dom.getDocumentElement().getAttribute("xmlns:inspire_vs"));
```

Comment:
@bgsmase are there any suitable locations to make asserts against the schemaLocation? This would confirm that your desired behaviour is implemented and help ensure that it is retained in future. Need test coverage!